plugin-server/src/main/ingestion-queues/jobs-consumer.ts

```diff
@@ -75,6 +81,18 @@ export const startJobsConsumer = async ({
                 continue
             }
 
+            if (skippedPluginConfigIds.includes(job.pluginConfigId.toString())) {
+                status.info('⬆️', 'Skipping plugin job because of config', {
```

Comment:
nit: This would be a little bit more clear, when I read the log message in the future
```suggestion
                status.info('⬆️', 'Skipping plugin job because config ID is blacklisted', {
```